### PR TITLE
Add rasppi install instructions

### PR DESCRIPTION
**Pull request by cniweb**
Signed-off-by: Christian Haeussler <c-n-i@web.de> (github: cniweb)

### CODE REVIEW

File: installation/rasppi.md [hunk 1]
```diff
@@ -6,5 +6,87 @@ layout: documentation
 
 # Installation on a Raspberry Pi
 
-to be done...
+There’s no pre-configured image for OpenHAB, so installation is done the old fashioned way via a command line.
```
Comment (kaikreuzer) [inline installation/rasppi.md]: Note: the correct spelling is "openHAB", not "OpenHAB"!

File: installation/rasppi.md [hunk 2]
```diff
@@ -6,5 +6,87 @@ layout: documentation
...
+`sudo raspi-config`
+
+Expand the filesystem; and from the advanced menu, change the memory split to 16. When are done, restart, and as good practice, run a full update
+
+`sudo apt-get update
+sudo apt-get upgrade`
+
+The easier way to install the OpenHAB runtime is via apt-get, but first we need to add a secure key and the new repository:
+
+`wget -qO - 'https://bintray.com/user/downloadSubjectPublicKey?username=openhab' |sudo apt-key add -
```
Comment (kaikreuzer) [inline installation/rasppi.md]: Does a single escaping work as expected for code blocks? Did you check the Jeykll-rendered output?

File: installation/rasppi.md [hunk 3]
```diff
@@ -6,5 +6,87 @@ layout: documentation
...
+Start with the latest (full) Raspbian SD image (not the “lite” version, these don’t include the Java Virtual Machine). Get your network cable plugged in, then boot up, and navigate through SSH. Run:
+
+`sudo raspi-config`
+
+Expand the filesystem; and from the advanced menu, change the memory split to 16. When are done, restart, and as good practice, run a full update
+
+`sudo apt-get update
+sudo apt-get upgrade`
+
+The easier way to install the OpenHAB runtime is via apt-get, but first we need to add a secure key and the new repository:
```
Comment (kaikreuzer) [inline installation/rasppi.md]: Please try not to duplicate information from http://docs.openhab.org/installation/apt.html, but rather reference it instead and keep on this page only the RaspPi specific stuff.

**Comment by cniweb** [comment]
Hi @kaikreuzer,

I implement your comments.

Chris

File: installation/rasppi.md [hunk 4]
```diff
@@ -6,5 +6,78 @@ layout: documentation
...
+
+Start with the latest (full) Raspbian SD image (not the “lite” version, these don’t include the Java Virtual Machine). Get your network cable plugged in, then boot up, and navigate through SSH. Run:
+
+`sudo raspi-config`
+
+Expand the filesystem; and from the advanced menu, change the memory split to 16. When are done, restart, and as good practice, run a full update
+
+Install openHAB on Linux, see [Installation on Linux through APT](http://docs.openhab.org/installation/apt.html)
+
+Curiously, everything was installed as owned by “root”. We need to fix that with the following commands.
```
Comment (kaikreuzer) [inline installation/rasppi.md]: @theoweiss Is this correct? I thought that APT uses the openhab user for the installation?

File: installation/rasppi.md [hunk 5]
```diff
@@ -6,5 +6,78 @@ layout: documentation
...
+
+Change the workgroup name if needed, but otherwise enable WINS support:
+
+`wins support = yes`
+
+(you’ll need to uncomment the line, and change no to yes)
+
+then add the following to the share definitions section (scroll all the way down to the bottom of the long file):
+
+`[OpenHAB Home]`
```
Comment (kaikreuzer) [inline installation/rasppi.md]: openHAB, not OpenHAB! Same on next line.

File: installation/rasppi.md [hunk 6]
```diff
@@ -6,5 +6,78 @@ layout: documentation
...
+ `comment= OpenHAB Home`
+ `path=/usr/share/openhab`
+ `browseable=Yes`
+ `writeable=Yes`
+ `only guest=no`
+ `create mask=0777`
+ `directory mask=0777`
+ `public=no`
+`[OpenHAB Config]`
```
Comment (kaikreuzer) [inline installation/rasppi.md]: dito

**Comment by kaikreuzer** [comment]
Thanks @cniweb, looks good! Just please adapt the spelling of openHAB :-)

**Comment by cniweb** [comment]
I adapt the spelling of openHAB!

**Comment by kaikreuzer** [comment]
Many thanks :-)

**Pull request merged**